get_predictions.py

```diff
@@ -57,14 +86,14 @@ def analyze_with_openai(image_path: str, verbose=False):
         "Authorization": f"Bearer {creds.OPENAI_API_KEY}",
     }
     payload = {
-        "model": "gpt-4-vision-preview",
+        "model": "gpt-4o",
         "messages": [
             {
                 "role": "user",
                 "content": [
                     {
                         "type": "text",
-                        "text": "Rate the road quality on a scale from 0 to 100. Your response should be in the form {ROAD_QUALITY: N}, where N is a number between 0 and 100. If the image does not contain a road, please enter 'NO_ROAD'. If the image is indoors, please enter 'INDOOR'. Start at a score of N=100. If the pavement quality looks rough, lower your score by 20. If the road contains some cracks, lower your score by 30. If the road contains a lot of cracks, lower your score by 40. If a road contains a pothole, lower your score by 80.",
+                        "text": PROMPT,
```

Comment:
question: does the new prompt outperform the previous one?

Reply:
It reduces rMAE and rMSE by ~5% but its still extremely noisy, the chain of thought had the most effect